### PR TITLE
fix: removing gcloud_skip_download variable

### PR DESCRIPTION
**Pull request by marko7460**
This PR fixes https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/issues/674

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by morgante — CHANGES_REQUESTED** [review]
Please make sure to remove from all submodules as well - https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/search?q=skip_download

**Comment by bharathkkb** [comment]
@morgante should we also add a sentence for discoverability linking to gcloud module on [how to use ](https://github.com/terraform-google-modules/terraform-google-gcloud#downloading)the env var? If we just include it in the upgrade guide, new users may not discover this is possible.

**Comment by morgante** [comment]
> @morgante should we also add a sentence for discoverability linking to gcloud module on how to use the env var? If we just include it in the upgrade guide, new users may not discover this is possible.

@bharathkkb Yes I think a note in the README would be a good idea.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ The resources/services/activations/deletions that this module will create/trigge
 
 Sub modules are provided for creating private clusters, beta private clusters, and beta public clusters as well.  Beta sub modules allow for the use of various GKE beta features. See the modules directory for the various sub modules.
 
+NOTE:<br>
+Some submodules use the [terraform-google-gcloud](https://github.com/terraform-google-modules/terraform-google-gcloud) module. By default, this module assumes you already have gcloud installed in your $PATH.
+See the [module](https://github.com/terraform-google-modules/terraform-google-gcloud#downloading) documentation for more information.
+
```
Comment (marko7460) [inline README.md:16]: @bharathkkb @morgante I added the note about the `gcloud` at the top of the README. Let me know if this is OK or if there should be a separate section.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ The resources/services/activations/deletions that this module will create/trigge
 
 Sub modules are provided for creating private clusters, beta private clusters, and beta public clusters as well.  Beta sub modules allow for the use of various GKE beta features. See the modules directory for the various sub modules.
 
+NOTE:<br>
+Some submodules use the [terraform-google-gcloud](https://github.com/terraform-google-modules/terraform-google-gcloud) module. By default, this module assumes you already have gcloud installed in your $PATH.
```
Comment (bharathkkb) [inline README.md:14]:
```suggestion
Some submodules use the [terraform-google-gcloud](https://github.com/terraform-google-modules/terraform-google-gcloud) module. By default, this module assumes you already have gcloud installed in your $PATH. If you would like to enable gcloud download and installation please set the environment variable as documented [here](https://github.com/terraform-google-modules/terraform-google-gcloud#downloading).
```

File: README.md (hunk 1 above)
Comment (bharathkkb) [reply README.md:16]: @marko7460 I think we can move this to the software dependencies section and have a sub heading there. Content LGTM
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine#software-dependencies

**Review by bharathkkb — APPROVED** [review]
Thanks @marko7460!

**Pull request merged**